language_server/src/lsp/feature_workspace_folders.cpp

```diff
@@ -19,6 +19,7 @@
 #include "network/uri/uri.hpp"
 
 #include "../logger.h"
+#include "config/pgm_conf.h"
```

Comment:
is this change still needed?

Reply:
Yes, because of the call on line 166 where `load_from_json` is called

Reply:
Curious, it definitely looks like it needs to be there, I wonder why the build was passing till now...

Reply:
I think it has something to do with type traits and I've tracked it to this release: https://github.com/nlohmann/json/releases/tag/v3.7.0

Also this issue talks about something very similar too: 
https://github.com/nlohmann/json/issues/1996

Reply:
ok, I think I see what is happening... there must be some new conversion operator that is trying to convert the json to the pgm_conf structure... isn't there an option to disable implicit json->anything conversion?
The config/pgm_conf.h is the library internal include, so we should not use it in the server source directly.
Renaming the method to load_from_pgm_config seems preferable to prevent the conversion attempt.

Reply:
I didn't find any suitable option -> function renamed